js/main.js

```diff
@@ -0,0 +1,50 @@
+(function() {
+  // All code NOT referencing DOM elements can go here
+  var title = document.querySelector("[data-js='title']");
```

Comment:
Said a similar thing on #1 - since this is referencing the `document` object, it should be code inside of the _DOMContentLoaded_ listener